### PR TITLE
Ironwood docs

### PR DESCRIPTION
**Pull request by nedbat**
## [DOC-XXXX](https://openedx.atlassian.net/browse/DOC-XXXX)

Add a description of your changes with links to any relevant material.

### Date Needed (optional)

If the release date of a feature is known or estimated, provide it to give reviewers guidance on turnaround time.

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [ ] Subject matter expert: 
- [ ] Subject matter expert: 
- [ ] Doc team review (sanity check, copy edit, or dev edit?): @edx/doc
- [ ] Product review:
- [ ] Partner support: 
- [ ] PM review: 

FYI: Tag anyone else who might be interested in this PR here.

### Testing

- [ ] Ran ./run_tests.sh without warnings or errors

### HTML Version (optional)

- [ ] Build an RTD draft for your branch and add a link here

### Sandbox (optional)

- [ ] Point to or build a sandbox for the software change and add a link here

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

**Comment by nedbat** [comment]
@grantgoodmanedX I think I did all the things I need to.  I really really want to get this published Thursday.  Ping me directly on Slack if there's anything I need to do to make it happen.

(Sorry for the short notice)

The release notes were written by Product in this Google Doc: https://docs.google.com/document/d/16X1-BIHffcNr_8lzKGEQ6qLKTktfdd_c8UQVSQ6lF0w/edit?ts=5c8916aa#  I adapted them.

File: en_us/install_operations/source/platform_releases/ironwood.rst [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+.. contents::
+ :local:
+ :depth: 1
+
+****************************
+What's Included in Ironwood
+****************************
+
+The Open edX Hawthorn release contains several new features for learners,
```
Comment (grantgoodmanedX) [inline en_us/install_operations/source/platform_releases/ironwood.rst]: Hawthorn --> Ironwood

File: en_us/install_operations/source/platform_releases/ironwood.rst [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+Occasionally, we release updates to Ironwood.  For example, the second
+release of Ironwood will be ``open-release/ironwood.2``.
+The steps to upgrade differ based on your original installation method.
+
+================================
+Upgrading a Docker Installation
+================================
+
+Devstack is installed using Docker. To upgrade from one Hawthorn
```
Comment (grantgoodmanedX) [inline en_us/install_operations/source/platform_releases/ironwood.rst]: Hawthorn --> Ironwood

File: en_us/open_edx_release_notes/source/ironwood.rst [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+programs through new program fragment views of the web edX platform pages. This
+new area enables learners to view their program dashboard and program progress
+pages, just as they do on the web browser experience.  
+
+With version 2.17 we delivered many accessibility improvements into the mobile
+applications, including initial support for iOS 11’s recently introduced
+Dynamic Text Type sizing and scaling. This change means learners who have set
+their device text size to being larger now have that setting expressed within
+the open edX mobile application. Additionally, the mobile application now
+provides clarity into when learner’s will lose access to a course. More details
```
Comment (grantgoodmanedX) [inline en_us/open_edx_release_notes/source/ironwood.rst]: learners

File: en_us/open_edx_release_notes/source/ironwood.rst [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+around this are included in the Feature Based Enrollments update included
```
Comment (grantgoodmanedX) [inline en_us/open_edx_release_notes/source/ironwood.rst]: It's not below. Either strike "included below" or add an anchor and link up to it.

File: en_us/open_edx_release_notes/source/ironwood.rst [hunk 5]
```diff
@@ -0,0 +1,276 @@
...
+rules or logic for password complexity.
+
+
+Expanded Language Support for course experience
+-----------------------------------------------
+
+We have added support for beta languages on the platform. These languages
+become a selectable option in the account settings dropdown alongside fully
+supported languages. If a learner selects a beta language they are shown a
+message letting them know support for this language is partial and makes to
```
Comment (grantgoodmanedX) [inline en_us/open_edx_release_notes/source/ironwood.rst]: "makes it easy to revert"

File: en_us/open_edx_release_notes/source/ironwood.rst [hunk 6]
```diff
@@ -0,0 +1,276 @@
...
+can now also provide your own password validator plugin if you have custom
+rules or logic for password complexity.
+
+
+Expanded Language Support for course experience
+-----------------------------------------------
+
+We have added support for beta languages on the platform. These languages
+become a selectable option in the account settings dropdown alongside fully
+supported languages. If a learner selects a beta language they are shown a
```
Comment (grantgoodmanedX) [inline en_us/open_edx_release_notes/source/ironwood.rst]: add comma after language

File: en_us/open_edx_release_notes/source/ironwood.rst [hunk 7]
```diff
@@ -0,0 +1,276 @@
...
+public outline, or public. 
+
+* Private (default, same as today)
+* Public Outline (redirects learners to the course outline page and shows the outline only)
+* Public (lets learners view HTML & Video content in the course, but hides unsupported XBlocks - such as problems and inline discussions)
+
+Feature Based Enrollments (FBE)
+-------------------------------
+
+Feature Based Enrollments, the basis of edX.org’s new revenue model - is
```
Comment (grantgoodmanedX) [inline en_us/open_edx_release_notes/source/ironwood.rst]: change "model - " to "model, "

**Review by grantgoodmanedX — CHANGES_REQUESTED** [review]
A few cases where Hawthorn needs to be changed to Ironwood, and a few miscellaneous style edits.

**Comment by nedbat** [comment]
Comments fixed.

**Pull request merged**